### PR TITLE
New sample disposal emails.

### PR DESCRIPTION
**Pull request by EGCGDanForth**
One script emails the Facility Manager to notify them that they should log into the LIMS and review samples for disposal.
One email notifies the Project Manager and Lead Sequencing technician that the Facility Manager has approved samples for disposal.
Tests added

### CODE REVIEW

File: scripts/email_sample_disposal_notification.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        )
+        subject = 'Samples Approved For Disposal'
+
+        # Send email to list of persons specified in the projects-facility-lab-finance_only section of config
+        self.send_mail(subject, msg, config_name='projects-lab')
+
+
+def main():
+    # Ge the default command line options
+    p = step_argparser()
```
Comment (mwhamgenomics) [inline scripts/email_sample_disposal_notification.py]: This needs to use the new command parsing:

```python
if __name__ == '__main__':
    SampleDisposalNotificationEmail().run()
```

File: scripts/email_sample_disposal_review.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        )
+        subject = 'Review Samples for Disposal'
+
+        # Send email to list of persons specified in the projects-facility-lab-finance_only section of config
+        self.send_mail(subject, msg, config_name='projects-facility')
+
+
+def main():
+    # Ge the default command line options
+    p = step_argparser()
```
Comment (mwhamgenomics) [inline scripts/email_sample_disposal_review.py]: Same as above

File: scripts/email_sample_disposal_notification.py (hunk 1 above)
Comment (EGCGDanForth) [reply scripts/email_sample_disposal_notification.py]: updated as requested

File: scripts/email_sample_disposal_notification.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+            project=self.projects[0].name,
+        )
+        subject = 'Samples Approved For Disposal'
+
+        # Send email to list of persons specified in the projects-facility-lab-finance_only section of config
+        self.send_mail(subject, msg, config_name='projects-lab')
+
+
+if __name__ == '__main__':
+    ReceiveSampleEmail().run()
```
Comment (mwhamgenomics) [inline scripts/email_sample_disposal_notification.py]: Unresolved reference - incorrect class name

File: scripts/email_sample_disposal_review.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+            project=self.projects[0].name,
+        )
+        subject = 'Review Samples for Disposal'
+
+        # Send email to list of persons specified in the projects-facility-lab-finance_only section of config
+        self.send_mail(subject, msg, config_name='projects-facility')
+
+
+if __name__ == '__main__':
+    ReceiveSampleEmail().run()
```
Comment (mwhamgenomics) [inline scripts/email_sample_disposal_review.py]: Same as above

Comment (EGCGDanForth) [reply scripts/email_sample_disposal_review.py]: updated with correct class

File: scripts/email_sample_disposal_notification.py (hunk 3 above)
Comment (EGCGDanForth) [reply scripts/email_sample_disposal_notification.py]: updated with correct class

File: scripts/email_sample_disposal_notification.py [hunk 5]
```diff
@@ -28,4 +28,4 @@ def _run(self):
 
 
 if __name__ == '__main__':
-    ReceiveSampleEmail().run()
+    SampleDisposalNotificationEmailEmail().run()
```
Comment (mwhamgenomics) [inline scripts/email_sample_disposal_notification.py]: Please.

**Comment by EGCGDanForth** [comment]
LIMS CR757

**Pull request merged**